configs/test.py

```diff
@@ -0,0 +1,28 @@
+from gem5.resources.resource import SuiteResource
+
+suite = {
+    "id": "id",
+    "resource_version": "1.0.0",
+    "gem5_versions": ["23.1"],
+    "workloads": [
+        {
+            "id": "riscv-ubuntu-20.04-boot",
+            "resource_version": "1.0.0",
+            "input_group": ["hashiru", "valorant"],
+        },
+        {
+            "id": "x86-ubuntu-18.04-boot",
+            "resource_version": "1.0.0",
+            "input_group": ["harshil", "valorant"],
+        },
+    ],
+}
+
+s = SuiteResource(
+    workloads=suite["workloads"],
+    resource_version=suite["resource_version"],
+)
+
+for workload in s.with_input_group("valorant"):
+    print(1)
```

Comment:
What is this print statement for?

Reply:
This was just for checking if it gets the correct number of workloads, this is not a proper test.